handler/url.go

```diff
@@ -59,8 +60,26 @@ func (h URL) Retrieve(c echo.Context) error {
 	return c.Redirect(http.StatusFound, url)
 }
 
+func (h URL) Count(c echo.Context) error {
+	ctx := c.Request().Context()
+
+	key := c.Param("url")
+
+	m := h.Store.(*store.MongoURL)
```

Comment:
:scream: Define count method on the store interface not casting the interface into a concrete type :scream: